docs/docs/rest-api/rest-json/rest-json-notebook-list-response.json

```diff
@@ -0,0 +1 @@
+{"status":"OK","message":"","body":[{"name":"Homepage","id":"2AV4WUEMK","config":{"looknfeel":"report"},"info":{}},{"name":"Zeppelin Tutorial","id":"2A94M5J1Z","config":{"looknfeel":"default"},"info":{}},{"name":"hive demo","id":"2B1H13MQG","config":{"looknfeel":"default","cron":"1"},"info":{"cron":"CronExpression \u00271\u0027 is invalid."}},{"name":"test dispaly","id":"2B2RSPF4A","config":{"looknfeel":"default","cron":"0 0 0 * * ?"},"info":{}}]}
```

Comment:
Yes, I deliberately left the error in the JSON to demonstrate how it will
look like.
Eran


On Mon, Oct 26, 2015 at 1:33 AM Felix Cheung <notifications@github.com>
wrote:

> In docs/docs/rest-api/rest-json/rest-json-notebook-list-response.json
> <https://github.com/apache/incubator-zeppelin/pull/367#discussion_r42952259>
> :
>
> > @@ -0,0 +1 @@
> > +{"status":"OK","message":"","body":[{"name":"Homepage","id":"2AV4WUEMK","config":{"looknfeel":"report"},"info":{}},{"name":"Zeppelin Tutorial","id":"2A94M5J1Z","config":{"looknfeel":"default"},"info":{}},{"name":"hive demo","id":"2B1H13MQG","config":{"looknfeel":"default","cron":"1"},"info":{"cron":"CronExpression \u00271\u0027 is invalid."}},{"name":"test dispaly","id":"2B2RSPF4A","config":{"looknfeel":"default","cron":"0 0 0 * * ?"},"info":{}}]}
>
> i guess you are intentionally showing the error msg.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-zeppelin/pull/367/files#r42952259>.
>
-- 
Eran | "You don't need eyes to see, you need vision" (Faithless)